ivadomed/config/example_loader_v2_configs.py

```diff
@@ -0,0 +1,398 @@
+                          "sub-05/ses-03/anat/sub-05_ses-03_flip-1_mt-on_MTS.nii",
+                          "sub-05/ses-03/anat/sub-05_ses-03_flip-1_mt-on_MTS.nii"],
+                         ["derivatives/labels/sub-05/ses-03/anat/sub-05_ses-03_mt-off_MTS_lesion-manual-rater1.nii"]],
+                        [["sub-05/ses-04/anat/sub-05_ses-04_flip-1_mt-off_MTS.nii", ],
+                         ["derivatives/labels/sub-05/ses-04/anat/sub-05_ses-04_mt-off_MTS_lesion-manual-rater1.nii"]],
+                    ],
+                    DataloaderKW.EXPECTED_INPUT: 2,
+                    DataloaderKW.EXPECTED_GT: 1,
+                    DataloaderKW.MISSING_FILES_HANDLE: FileMissingHandleKW.SKIP,
+                    DataloaderKW.EXCESSIVE_FILES_HANDLE: FileExcessiveHandleKW.USE_FIRST_AND_WARN,
+                    DataloaderKW.PATH_DATA: path_mock_data,
+                }
+            ],
+            DataloaderKW.TEST: [
+                {
+                    DataloaderKW.TYPE: "FILES",
+                    DataloaderKW.SUBSET_LABEL: "TestFileDataSet1",
+                    DataloaderKW.IMAGE_GROUND_TRUTH: [
+                        [["sub-06/ses-01/anat/sub-06_ses-01_flip-1_mt-off_MTS.nii",
+                          "sub-06/ses-01/anat/sub-06_ses-01_flip-1_mt-on_MTS.nii"],
+                         ["derivatives/labels/sub-06/ses-01/anat/sub-06_ses-01_mt-off_MTS_lesion-manual-rater1.nii"]],
+                        [["sub-06/ses-02/anat/sub-06_ses-02_flip-1_mt-off_MTS.nii",
+                          "sub-06/ses-02/anat/sub-06_ses-02_flip-1_mt-on_MTS.nii"],
+                         ["derivatives/labels/sub-06/ses-02/anat/sub-06_ses-02_mt-off_MTS_lesion-manual-rater1.nii"]],
+                        [["sub-06/ses-03/anat/sub-06_ses-03_flip-1_mt-off_MTS.nii",
+                          "sub-06/ses-03/anat/sub-06_ses-03_flip-1_mt-on_MTS.nii",
+                          "sub-06/ses-03/anat/sub-06_ses-03_flip-1_mt-on_MTS.nii"],
+                         ["derivatives/labels/sub-06/ses-03/anat/sub-06_ses-03_mt-off_MTS_lesion-manual-rater1.nii"]],
+                        [["sub-06/ses-04/anat/sub-06_ses-04_flip-1_mt-off_MTS.nii", ],
+                         ["derivatives/labels/sub-06/ses-04/anat/sub-06_ses-04_mt-off_MTS_lesion-manual-rater1.nii"]],
+                    ],
+                    DataloaderKW.EXPECTED_INPUT: 2,
+                    DataloaderKW.EXPECTED_GT: 1,
+                    DataloaderKW.MISSING_FILES_HANDLE: FileMissingHandleKW.SKIP,
+                    DataloaderKW.EXCESSIVE_FILES_HANDLE: FileExcessiveHandleKW.USE_FIRST_AND_WARN,
+                    DataloaderKW.PATH_DATA: path_mock_data,
+                }
+            ],
+            DataloaderKW.EXPECTED_INPUT: 2,
+            DataloaderKW.EXPECTED_GT: 1,
+        },
+    ],
+    DataloaderKW.EXPECTED_INPUT: 2,
+    DataloaderKW.EXPECTED_GT: 1,
+}
+
+example_1i1o_all_dataset_groups_config_json: dict = {
```

Comment:
Just for my understanding:

2i1o above is the multichannel case.
1i10 here is a single channel case, and when 2 subject files (inputs) are provided instead of 1, the second one is discarded.

Is that right?

Reply:
Absolutely. Perfectly correct.